unit/BasicNativeAndroidTest/app/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+apply plugin: 'kotlin-android'
+
+android {
```

Comment:
I haven't gotten around to filing a bug for this, but consumers of junit-gtest actually need to exclude some cruft that's included in the AAR or it will end up in the APK:  https://github.com/DanAlbert/ndk-app-template/blob/6f46d2f49e68d20c102eb2122262716564f191d2/app/build.gradle#L42-L48 (in this example you don't want to exclude libc++_shared since this build is using libc++_shared; the example I linked uses the static lib)

Not sure if you want to include that workaround in this example, or fix the bug before submitting this. I'm fine either way.

The rest LGTM though.